src/components/PageSizer.js

```diff
@@ -1,23 +1,29 @@
 import React, { useCallback } from 'react'
 import propTypes from 'prop-types'
+import { useManifest } from 'use-manifest'
+
 import useUrlParamState from '../hooks/useUrlParamState'
 
-const DEFAULT_PAGE_SIZES = [10, 20, 50, 100, 200]
+const DEFAULT_PAGE_SIZES = [10, 20, 50, 100]
 
 const DEFAULT_TEXT_GENERATOR = size => `Show ${size} entries`
 
+const numbersAscending = (a, b) => a - b
```

Comment:
I think this is the default behavior of `arr.sort()`.  See if this will work correctly with no sort function provided.

Reply:
It is not, for some reason it sorts them... alphabetically i guess
(1, 100, 2, 22, 3)

Reply:
That is strange, but yes I verified that on my side too.